src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers.csproj

```diff
@@ -19,7 +13,8 @@
 
   <Import Project="..\..\TargetFrameworkDependentValues.props" />
 
-  <PropertyGroup>
+  <!-- Only use $(ToolOutputFullPath) for netstandard2.0 -->
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <OutputPath>$(ToolOutputFullPath)</OutputPath>
   </PropertyGroup>
```

Comment:
I think what I ended up with is simple. This prevents us from putting the `net8.0` library in `$(ToolOutputFullPath)`.

The build error is gone in https://github.com/xamarin/xamarin-android/pull/8751, but we still have some runtime test failures.